addons/udes_permissions/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "udes_permissions",
+    "summary": """
```

Comment:
Q: Do we need both summary and description? I can't remember... will come back in the morning. :P

Could also borrow a few words from the readme to use.